ground/src/main/java/com/google/android/ground/persistence/sync/MediaUploadWorker.kt

```diff
@@ -133,23 +94,10 @@ constructor(
         throw FileNotFoundException(photoFile.path)
       }
       remoteStorageManager.uploadMediaFromFile(photoFile, path)
-      kotlin.Result.success(Unit)
-    } catch (t: FirebaseNetworkException) {
-      Timber.d(t, "Can't connect to Firebase to upload photo")
-      kotlin.Result.failure(t)
+      return kotlin.Result.success(Unit)
     } catch (t: Throwable) {
-      Timber.e(t, "Failed to upload photo")
-      kotlin.Result.failure(t)
+      Timber.log(t.priority(), t, "Photo upload failed")
+      return kotlin.Result.failure(t)
     }
-
-  companion object {
-    private const val LOI_ID = "locationOfInterestId"
-
-    @JvmStatic
-    fun createInputData(locationOfInterestId: String): Data =
-      Data.Builder().putString(LOI_ID, locationOfInterestId).build()
-
-    private fun SubmissionMutation.canRetry() =
-      this.retryCount < Config.MAX_MEDIA_UPLOAD_RETRY_COUNT
```

Comment:
Let's cleanup the unused constant as well from `Config.kt`

Reply:
Done!